component/azstorage/azstorage_constants.go

```diff
@@ -60,3 +60,14 @@ const (
 	size        = "Size"
 	target      = "Target"
 )
+
+// headers which should be logged and not redacted
+var allowedHeaders []string = []string{
+	"x-ms-version", "x-ms-date", "x-ms-range", "x-ms-delete-snapshots", "x-ms-delete-type-permanent", "x-ms-blob-content-type",
```

Comment:
request ids ? they will be important for backend tracing.

Reply:
Request ids are logged by default.